benchmarks/ReactiveUI.Benchmarks/AutoPersistBenchmark.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Reactive;
+using System.Reactive.Linq;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Attributes.Exporters;
+using BenchmarkDotNet.Attributes.Jobs;
+
+namespace ReactiveUI.Benchmarks
+{
+    [CoreJob]
+    [MarkdownExporterAttribute.GitHub]
+    public class AutoPersistBenchmark
+    {
+        private ReactiveList<string> _collection;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _collection = new ReactiveList<string>(new[]
+            {
+                "ReactiveUI",
+                "ReactiveUI.XamForms",
+                "ReactiveUI.WPF",
+                "ReactiveUI.Events"
+            });
+        }
+
+        [Benchmark]
```

Comment:
I wonder if there is any penalty having the xunit fact in this file. I imagine that harness file getting pretty large

Reply:
Because it's xUnit and it should simply run all `[Fact]` in the project, we should be able to break up the harness files into smaller files in the future as needed.